lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -187,6 +234,16 @@ def get_isEnabled(self, obj):
         """
         return obj['isEnabled'] and not obj['oraMetadata'].teams_enabled
 
+class AssessmentFeedbackSerializer(serializers.Serializer):
+    """
+    Serialize info for the initialize call. Packages ORA, course, submission, and rubric data.
+    """
```

Comment:
Can we update this docstring with a better description? https://peps.python.org/pep-0257/